subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_agps

```
@@ -3,8 +3,39 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-config NRF_CLOUD_AGPS
+menuconfig NRF_CLOUD_AGPS
 	bool "nRF Cloud Assisted GPS (A-GPS)"
 	depends on MODEM_INFO
 	depends on MODEM_INFO_ADD_NETWORK
 	select CJSON_LIB
+
+if NRF_CLOUD_AGPS
+
+config NRF_CLOUD_AGPS_DISABLE_SCHEDULED_DOWNLOADS
+	bool "Do not keep GNSS unit on to periodically download from satellites"
+	help
+	  For mfw_1.3.1 or greater, the GNSS unit will periodically stay on
+	  longer than normal and download satellite data. Disabling this
+	  reduces power consumption when it is known that A-GPS has already
+	  injected all required assistance data.
+
+config NRF_CLOUD_AGPS_FILTERED
+	bool "Request only visible satellite ephemerides"
+	select NRF_CLOUD_AGPS_DISABLE_SCHEDULED_DOWNLOADS
+	help
+	  Normally A-GPS requests ephemerides for all GPS constellation
+	  satellites. Setting this option reduces that set to only
+	  satellites at or above the threshold elevation angle.
+
+config NRF_CLOUD_AGPS_ELEVATION_MASK
+	int "Minimum elevation angle for visible satellites"
+	default 5
+	depends on NRF_CLOUD_AGPS_FILTERED
+	help
+	  This sets the threshold elevation angle in A-GPS cloud requests.
+	  It constrains which satellite ephemerides are included in the
+	  assistance data returned by the cloud. This value will also be used
+	  by the GNSS unit to decide which satellites to include when calculating
+	  a GPS position fix.
```

This option only controls the data download, GNSS needs to be configured by the application.
```suggestion
	  This sets the threshold elevation angle in A-GPS cloud requests.
	  It constrains which satellite ephemerides are included in the
	  assistance data returned by the cloud.
```